appcues/src/main/java/com/appcues/trait/appcues/TooltipTrait.kt

```diff
@@ -101,8 +100,10 @@ internal class TooltipTrait(
         val density = LocalDensity.current
         val metadata = LocalAppcuesStepMetadata.current
 
+        // trait does not apply if no target rectangle in metadata
+        val targetRectInfo = rememberTargetRectangleInfo(metadata) ?: return
```

Comment:
so in the case target rect is not available I had set to show the tooltip was preferred position NONE, which position the tooltip at the bottom of the view. wondering if now that we have a more defined behavior if we should remove that entirely

Reply:
right, I need to review this part again, I missed that and it should not just ignore the trait here.

Reply:
reverted this and tested it out - no target rectangle or element, and no backdrop-keyhole - just the default backdrop

It seems pretty close, some sort of sizing issue clipping off the bottom.

![Screenshot 2023-03-08 at 9 06 24 AM](https://user-images.githubusercontent.com/19266448/223733825-cd14a914-d121-4e8f-af99-fde4ae754caf.png)